nano/node/active_transactions.cpp

```diff
@@ -20,10 +20,7 @@ thread ([this]() {
 })
 {
 	std::unique_lock<std::mutex> lock (mutex);
-	while (!started)
-	{
-		condition.wait (lock);
-	}
+	condition.wait (lock, [this] { return this->started; });
```

Comment:
Try and only capture what you need to in lambdas otherwise it has a greater dependency than it needs, so in this case `&started = started` would be enough. And same in other places